libsplinter/src/biome/oauth/store/mod.rs

```diff
@@ -130,8 +130,8 @@ impl OAuthUserBuilder {
     /// Set the OAuth refresh token.
     ///
     /// This field is optional when constructing the final struct.
-    pub fn with_refresh_token(mut self, refresh_token: String) -> Self {
-        self.refresh_token = Some(refresh_token);
+    pub fn with_refresh_token(mut self, refresh_token: Option<String>) -> Self {
```

Comment:
What's the reason for this change?

Reply:
This allows the refresh token to be unset.

Reply:
I could see this being in the wrong commit though, will split it out.

Reply:
Split into it's own commit